docs/sandbox/stacking-context-6.html

```diff
@@ -2,7 +2,7 @@
   <head>
     <meta charset="utf-8">
     <title>5. Stacking Context example 1</title>
```

Comment:
It seems you forgot to change the title when duplicating/copy-pasting from example 5.